validation-test/stdlib/Lazy.swift.gyb

```diff
@@ -241,27 +240,26 @@ LazyTestSuite.test("EmptyCollection/AssociatedTypes") {
   typealias Subject = EmptyCollection<OpaqueValue<Int>>
   expectRandomAccessCollectionAssociatedTypes(
     collectionType: Subject.self,
-    iteratorType: EmptyIterator<OpaqueValue<Int>>.self,
+    iteratorType: EmptyCollection<OpaqueValue<Int>>.Iterator.self,
     subSequenceType: Subject.self,
     indexType: Int.self,
     indicesType: Range<Int>.self)
 }
 
-% for (name, operation) in [
-%   ('index(after:)', '_ = c.index(after: i)'),
-%   ('index(before:)', '_ = c.index(before: i)'),
-%   ('subscript(Index)/Get', '_ = c[i]'),
-%   ('subscript(Index)/Set', 'c[i] = OpaqueValue(42)'),
-% ]:
-LazyTestSuite.test("EmptyCollection/${name}")
-  .forEach(in: [-1, 0, 1]) {
-  i in
-
-  var c = EmptyCollection<OpaqueValue<Int>>()
-  expectCrashLater()
-  ${operation}
+let emptyCollectionIndexTests: [(String,(inout EmptyCollection<OpaqueValue<Int>>,Int)->Void)] = [
```

Comment:
s/Void/() would make the line shorter =)

Reply:
I think `()` for arguments, `Void` for return is our style tho.